plugin/db/driver.go

```diff
@@ -233,6 +233,13 @@ type MigrationInfo struct {
 	IssueID        string
 	Payload        string
 	CreateDatabase bool
+	// UseSemanticVersion is whether version is a semantic version.
+	// When UseSemanticVersion is set, version should be set to the format specified in Semantic Versioning 2.0.0 (https://semver.org/).
+	// For example, for settings non-semantic version "hello", the values should be Version = "hello", UseSemanticVersion = false, SemanticVersionSuffix = "".
```

Comment:
```suggestion
	// For example, for setting non-semantic version "hello", the values should be Version = "hello", UseSemanticVersion = false, SemanticVersionSuffix = "".
```

Reply:
Would be better to comment on the stored version format.

Reply:
Yeah, I was thinking to link to the design doc once we publish it.